CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# ooc-kean changes by release
+
+## Release-x.x.x
+
+* Example 0
+	Explanation of Example 0
```

Comment:
I suggest we use this format instead:
- Example 0
  - Explanation of example 0